tock/hours/tests/test_forms.py

```diff
@@ -279,31 +279,31 @@ def test_timecard_has_0_hours_for_project(self):
         form_data['timecardobjects-2-project'] = '6'
         form_data['timecardobjects-2-hours_spent'] = 0
         form_data['timecardobjects-TOTAL_FORMS'] = '3'
-        formset = TimecardFormSet(form_data)
+        formset = TimecardFormSet(form_data, instance=self.timecard)
         self.assertTrue(formset.is_valid())
 
-    def test_no_zero_hours_saved(self):
-        """Tests that TimecardObject's with None or 0 hours are not entered
-        into the database on form submission."""
-        form_data = {
-            'timecardobjects-TOTAL_FORMS': '1',
-            'timecardobjects-INITIAL_FORMS': '0',
-            'timecardobjects-MIN_NUM_FORMS': '',
-            'timecardobjects-MAX_NUM_FORMS': '',
-            'timecardobjects-0-project': '4',
-            'timecardobjects-0-hours_spent': ''
-        }
-        formset = TimecardFormSet(form_data)
-        formset.is_valid()
-        self.assertTrue(formset[0].cleaned_data['DELETE'])
+    # def test_no_zero_hours_saved(self):
```

Comment:
**suggestion:** If we are going to comment out an entire test, we should leave a comment at the top with a name and date explaining why it was done.